features/reconciliate_products.feature

```diff
@@ -0,0 +1,36 @@
+@reconcile_products
+Feature: Reconcile products
+  In order to conciliate my products in the store with the products from Akeneo
+  As a Store Owner
+  I want to reconcile them
+
+  Background:
+    Given the store operates on a single channel in "United States"
+
+  @cli
+  Scenario: Reconcile simple products
+    Given there is 1 product on Akeneo
+    And the store has a product "product-1"
+    And the store has a product "product-2"
+    When I reconcile items
+    Then the "product-1" product is enabled
+    And the "product-2" product is disabled
+
+  @cli
+  Scenario: Reconcile configurable products
+    Given there is a product "product-1-variant-1" updated at "2021-04-19" on Akeneo
+    And there is a product "product-1-variant-2" updated at "2021-04-19" on Akeneo
+    And there is a product "product-2-variant-2" updated at "2021-04-19" on Akeneo
+    And the store has a product "product-1-variant-1"
+    And this product has "product-1-variant-2" variant priced at "$25"
+    And the store has a product "product-2-variant-1"
+    And this product has "product-2-variant-2" variant priced at "$25"
+    And the store has a product "product-3-variant-1"
+    And this product has "product-3-variant-2" variant priced at "$25"
+    When I reconcile items
+    Then the "product-1-variant-1" product variant is enabled
+    Then the "product-1-variant-2" product variant is enabled
+    And the "product-2-variant-1" product variant is disabled
+    And the "product-2-variant-2" product variant is enabled
+    And the "product-3-variant-1" product variant is disabled
+    And the "product-3-variant-2" product variant is disabled
```

Comment:
This also checks if the `product-3-variant-1` parent product is disabled?

Reply:
The `@Given /^the ("([^"]*)" product) is disabled$/` contains an assertion to verify that product has only one variant. I can check if there is another way to add assertion also on parent product status.

Reply:
But the `product-3-variant-1` has two variants, right? I mean `product-3-variant-1` itself and the `product-3-variant-2`. If so, why it pass if it asserts that there's only one variant? Anyway I think we should also check that the parent product is disabled if there are no more enabled variants.

Reply:
In fact, it didn't pass, I tested it on my machine 😄 . Anyway, I refactored all the statements because they were context statements, not product status checks 🤦🏻 .